test/Array/array_sort3.js

```diff
@@ -116,19 +118,19 @@ write(arr.sort());
 
 write(arr);
 
-function comparefn(x,y) { arr[0]="test"; return x - y; }
+function comparefnOne(x,y) { arr[0]="test"; return x - y; }

```

Comment:
This test case had two identically named compare functions therefore hoisting meant only the second one was used. Renamed so both are used.